builtin-functions/kphp-light/unsupported/arrays.txt

```diff
@@ -65,7 +65,8 @@ function in_array ($value ::: any, $a ::: array, $strict ::: bool = false) ::: b
 function array_fill ($start_index ::: int, $num ::: int, $value ::: any) ::: ^3[];
 function array_fill_keys ($a ::: array, $value ::: any) ::: ^2[];
 function array_combine ($keys ::: array, $values ::: array) ::: ^2;
-function range ($from, $to, $step ::: int = 1) ::: mixed[];//TODO
+/** @kphp-extern-func-info generate-stub */
+function range ($from ::: mixed, $to ::: mixed, $step ::: int = 1) ::: mixed[];//TODO
```

Comment:
What does TODO comment mean here?
Please remove or provide better description

Reply:
I don't know. It came from _functions.txt. Remove it